salt/example.top

```diff
@@ -9,12 +9,13 @@ base:
         - elife.gearman
         - elife.golang
         - elife.aws-cli
-        - search.aws
         - elife.goaws
+        - elife.proofreader-php
+        - search.elasticsearch
+        - search.aws
         - search.java
+        - search.gearman
```

Comment:
no problem for changing the order but see https://github.com/elifesciences/builder-private/pull/40#issuecomment-328686354 in case `require_in` are going wild